src/dpr/utils/renderSyncReport.ts

```diff
@@ -0,0 +1,103 @@
+import { Request } from 'express'
+import { SyncReportFeatures, SyncReportOptions, SyncReportUtilsParams } from '../types/SyncReportUtils'
+import ReportQuery from '../types/ReportQuery'
+import SyncReportUtils from '../components/sync-report-list/utils'
+import { LoadType } from '../types/UserReports'
+import { Services } from '../types/Services'
+
+const initAdditionalFeatures = async (
+  features: SyncReportFeatures,
+  services: Services,
+  req: Request,
+  userId: string,
+) => {
+  const { reportId, id } = req.params
+
+  let canDownload
+  let bookmarked
+  let removeBookmark = true
+
+  if (features.download) {
+    canDownload = await services.downloadPermissionService.downloadEnabled(userId, reportId, id)
```

Comment:
Maybe if the downloadPermissionService hasn't been set up we could catch that here and give a meaningful message? Something like "The download feature has been enabled but the downloadPermissionService hasn't been configured. See the setup guide for further information: https://..."